build.sbt

```diff
@@ -1,6 +1,4 @@
-scalaVersion := "2.10.6"
-
-crossScalaVersions := Seq("2.10.6", "2.11.8")
+scalaVersion in ThisBuild := "2.11.8"
```

Comment:
@gianm Since build for scala 2.12 will lead to update of finagle/twitter version (builds for scala 2.12 starts from `6.41.0` version), hence we have to update mmx scala-util library. Do you think in this case it's better to close this PR and remove/add scala versions in #220 ?